docs/data/product/dea-tasseled-cap-percentiles-landsat/_details.md

```diff
@@ -4,34 +4,38 @@ The Tasseled Cap (Kauth&ndash;Thomas) transformation takes satellite imagery and
 
 ## What this product offers
 
-This product offers three percentiles (10th, 50th and 90th) per Tasseled Cap index (greeness, wetness and brightness) per year, from 1987 to the present. 
+This product offers three percentiles (10th, 50th and 90th) per Tasseled Cap index (greeness, wetness and brightness) per year, from 1987 to the present.
```

Comment:
```suggestion
This product offers three percentiles per Tasseled Cap index per year. The percentiles are 10th, 50th, and 90th; the Tasseled Cap indexes are 'greeness', 'wetness', and 'brightness'; and the years are 1987 to 2023.
```

Reply:
**Question:** Is it 'greeness' or 'greenness' since that word normally has two n's

Reply:
greenness is correct!